### PR TITLE
Publish Web Audio API v1.1 FPWD

### PR DESCRIPTION
**Pull request by hoch**
This PR is to publish FPWD of Web Audio API v1.1. The change includes followings:

- Update the version to 1.1 and the document stage to FPWD
- Collapse all the changes with mark-ups (proposed addition/correction) since Recommendation into the spec:
  - https://github.com/WebAudio/web-audio-api/issues/2361
  - https://github.com/WebAudio/web-audio-api/issues/127
  - https://github.com/WebAudio/web-audio-api/issues/2359
  - https://github.com/WebAudio/web-audio-api/issues/2373
  - https://github.com/WebAudio/web-audio-api/issues/2321
  - https://github.com/WebAudio/web-audio-api/issues/2375
  - https://github.com/WebAudio/web-audio-api/issues/2475
  - https://github.com/WebAudio/web-audio-api/issues/2444
  - https://github.com/WebAudio/web-audio-api/issues/2456
  - https://github.com/WebAudio/web-audio-api/issues/2400
  - https://github.com/WebAudio/web-audio-api/issues/2512
  - https://github.com/WebAudio/web-audio-api/issues/2450
  - https://github.com/WebAudio/web-audio-api/issues/2567
- Remove JS code that handles the mark-ups.
- Remove the content change log section

References
- This change is approved by W3C: https://github.com/w3c/transitions/issues/656
- This PR fixes https://github.com/WebAudio/web-audio-api/issues/2602

***
<a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/pull/2603.html" title="Last updated on Oct 3, 2024, 5:02 PM UTC (c92bc42)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/2603/89b51b1...c92bc42.html" title="Last updated on Oct 3, 2024, 5:02 PM UTC (c92bc42)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
 ED: https://webaudio.github.io/web-audio-api/
-TR: https://www.w3.org/TR/webaudio/
+TR: https://www.w3.org/TR/webaudio-11/
 Favicon: favicon.png
 Previous Version: https://www.w3.org/TR/2021/CR-webaudio-20210114/
```
Comment (mjwilson-google) [inline index.bs:10]: Should we add https://www.w3.org/TR/2021/REC-webaudio-20210617/ to previous versions too?

File: index.bs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+Title: Web Audio API 1.1
 Shortname: webaudio
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
 ED: https://webaudio.github.io/web-audio-api/
-TR: https://www.w3.org/TR/webaudio/
+TR: https://www.w3.org/TR/webaudio-11/
```
Comment (mjwilson-google) [inline index.bs:8]: I'm not sure how many Web Audio versions we're planning on, but should this perhaps be "webaudio-1-1" or "webaudio-1_1"?

Comment (hoch) [reply index.bs:8]: This URL is suggested by @svgeesus.

File: index.bs (hunk 1 above)
Comment (hoch) [reply index.bs:11]: Good thought! Done.

**Review by mjwilson-google — APPROVED** [review]
LGTM

File: index.bs [hunk 3]
```diff
@@ -1,12 +1,13 @@
 <pre class=metadata>
-Title: Web Audio API
+Title: Web Audio API 1.1
 Shortname: webaudio
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
```
Comment (padenot) [inline index.bs:6]: Shouldn't this stay `ED` for the main branch, and `FPWD` for a specific branch?

Comment (hoch) [reply index.bs:6]: I think when we discussed this matter in person, the resolution was to use the versioning to keep the latest REC with markups. (See https://github.com/WebAudio/web-audio-api/releases/tag/1.0-with-markups-tpac-2024)

Do we want to use separate branches to handle these? I think we should be able to specify the branch that we want to use for the GitHub landing, but we should discuss the logistics.

**Review by mdjp — APPROVED** [review]
Catching up a bit, but I am happy with this.

File: index.bs (hunk 3 above)
Comment (padenot) [reply index.bs:6]: It's more or less all branches (branches are just pointers to a commit in git). As long as the `main` branch is the `ED` and has the markup removed (so we can add more changes), and we have clear pointers to various steps we've done, I think we're good.

Comment (hoch) [reply index.bs:6]: I also prefer something simple, so we're aligned on that matter.

@svgeesus Should we use `ED` or `FPWD` for this meta entry?

Comment (svgeesus) [reply index.bs:6]: It is fine to use ED for clarity. I can change it to FPWD locally when publishing.

Comment (hoch) [reply index.bs]: Thanks! Done.

**Comment by hoch** [comment]
@padenot @svgeesus PTAL again!

**Review by svgeesus — APPROVED** [review]
This looks good to me

**Comment by svgeesus** [comment]
@padenot did you want to re-review, or ok to merge?

**Comment by svgeesus** [comment]
There are some [broken links](https://validator.w3.org/checklink?uri=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2FWebAudio%2Fweb-audio-api%2Fpull%2F2603.html&hide_type=all&depth=&check=Check):

- [https://www.w3.org/html/wg/drafts/2dcontext/html5_canvas_CR/](https://www.w3.org/html/wg/drafts/2dcontext/html5_canvas_CR/)
- https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/pull/2603.html#c2359
-  https://w3c.github.io/mediacapture-main/#dom-mediastreamtrack (lines 2058, 2789, 3157, 3160, 4434, 9276, 9357, 9385, 9386, 9387, 9400, 9404, 9414, 9465, 9490, 9536, 9541, 9543, 14570, 14979)
- https://w3c.github.io/mediacapture-main/#dom-mediastream (lines 2055, 2061, 2787, 3083, 3136, 3329, 9276, 9278, 9318, 9355, 9357, 9361, 9391, 9400, 9412, 9416, 9448, 9450, 9455, 9460, 14568, 14960, 14966, 14970)
- https://w3c.github.io/mediacapture-main/#dom-mediadevices (line 13069)
- https://w3c.github.io/mediacapture-main/#dom-mediadevices-enumeratedevices (line 3471)
- https://w3c.github.io/hr-time/#dom-domhighrestimestamp (lines 3548, 3557, 14592)

**Comment by hoch** [comment]
If that's okay, I'd like to keep this PR is about removing mark-ups. On a follow up PR, I can fix all the broken links.

**Comment by hoch** [comment]
Re @svgeesus:

- https://www.w3.org/html/wg/drafts/2dcontext/html5_canvas_CR/: [2dcontext] in [spec ref](https://www.specref.org/?q=2dcontext) is pointing an invalid link (404). Perhaps we can remove this link from the spec?
- https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/pull/2603.html#c2359: Fixed
- These links don't seem to be broken. Not sure what to do:
    - https://w3c.github.io/mediacapture-main/#dom-mediastreamtrack
    - https://w3c.github.io/mediacapture-main/#dom-mediastream
    - https://w3c.github.io/mediacapture-main/#dom-mediadevices
    - https://w3c.github.io/mediacapture-main/#dom-mediadevices-enumeratedevices
    - https://w3c.github.io/hr-time/#dom-domhighrestimestamp

**Comment by mjwilson-google** [comment]
> [2dcontext] in [spec ref](https://www.specref.org/?q=2dcontext) is pointing an invalid link (404). Perhaps we can remove this link from the spec?

It seems like this is now part of the HTML spec: https://html.spec.whatwg.org/multipage/canvas.html#2dcontext

Should we replace the reference with [[HTML]](https://www.specref.org/?q=https%3A%2F%2Fhtml.spec.whatwg.org%2Fmultipage%2F) instead?

**Comment by hoch** [comment]
I am not sure if that's a helpful change. If that's the only solution, linking "Canvas 2D" to the entire HTML spec is a bit misleading I guess?

I think the W3C link checker is a bit aggressive. It also complains about the document fragment (`#`). Not sure why this is not allowed.

**Comment by svgeesus** [comment]
> I think the W3C link checker is a bit aggressive. It also complains about the document fragment (`#`). Not sure why this is not allowed.

It complains if it can't find the specific fragment in that document.

However it doesn't run JS on fetched resources so is vulnerable to false negatives.

**Comment by svgeesus** [comment]
> I am not sure if that's a helpful change. If that's the only solution, linking "Canvas 2D" to the entire HTML spec is a bit misleading I guess?

We can add a local bibliography that defines [2DContext] and links to https://html.spec.whatwg.org/#2dcontext

specref should also be updated, but we don't have to be blocked on that.

**Pull request merged**